### ISSUE TITLE
Incorrect validation error after only changing the date

### ISSUE THREAD

**Issue by manderson23** [PredixDev/px-rangepicker#21]
I have a component configured as follows
```
<px-rangepicker id="datetimePicker" date-format="YYYY/MM/DD" time-format="HH:mm:ss" range="{{range}}" hide-presets="true"></px-rangepicker>
```
where the range is `2016/11/11 16:07:00` to `2016/11/11 16:10:00`

The following steps lead to an incorrect validation error:

1. Open the picker
2. Click 10th November. The range is correct flagged invalid as from > to.
3. Click 10th November a second time. The range is still marked as invalid even though from < to as shown below.
![screenshot from 2016-11-11 16-12-00](https://cloud.githubusercontent.com/assets/1684641/20221557/aaa553ac-a829-11e6-9116-306193043d81.png)

**Comment by benoitjchevalier** [comment]
Thanks for your feedback, we'll have to prioritize this issue and investigate

**Comment by Menkhaus-ge** [comment]
Hi, 
Thank you for helping us make Predix UI better. 
We have fixed this issue on v0.5.6 of px-calendar-picker. https://github.com/PredixDev/px-calendar-picker/releases/tag/v0.5.6

**Comment by manderson23** [comment]
Thanks
